app/src/main/res/values/strings.xml

```diff
@@ -15,4 +15,11 @@
     <string name="top_track">Top Track</string>
     <string name="search">Search</string>
     <string name="pcm_music">PCM Music</string>
+    <string name="singer_name_demo">Join Harry</string>
+    <string name="song_name_demo">Love You Like Love Songzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzz</string>
+    <string name="id_demo">1</string>
+    <string name="songs_count">5 bài hát</string>
+    <string name="artist_name">Join Harry</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.artist_name` appears to be unused
 Unused resources make applications larger and slow down builds.